main.py

```diff
@@ -314,6 +315,9 @@ def ui_access_wrapper(self, *args, **kwargs):
         reader.RecommendationsHandler),
 
     get_redirect_route(
+        r'/ml/trainedclassifierhandler', classifier.TrainedClassifierHandler),
```

Comment:
Let's put this at the bottom of the list since it starts with /ml/. It seems to fall in its own category, and shouldn't split the other "regular" URLs.